examples/with-react-ga4/utils/analytics.js

```diff
@@ -19,6 +19,9 @@ export const logEvent = (category = '', action = '') => {
 
 export const logException = (description = '', fatal = false) => {
   if (description) {
-    ReactGA.exception({ description, fatal })
+    ReactGA.gtag('event', 'exception', {
+      description: description,
+      fatal: fatal, // set to true if the error is fatal
```

Comment:
```suggestion
      description,
      fatal, // set to true if the error is fatal
```

These names are redundant